client/lib/redux-bridge/README.md

```diff
@@ -1,5 +1,5 @@
 A little bridge that consumes the current redux store and exposes the dispatch method as a singleton.
 
-This helps flux and thunk action handlers move to redux in an incremental fashion.
+This helps thunk action handlers move to redux in an incremental fashion.
```

Comment:
I don't see any connection between thunk action handlers and Redux Bridge. The use case for `lib/redux-bridge` at this moment is to provide access to Redux (`getState` or `dispatch`) to code that doesn't have the Redux store in any kind of context -- either React or page.js one.

Usages are currently quite rare (total ~10 of them) and quickly skimming over them I think most can be easily replaced.

Reply:
I left it as it was, but yeah, I think it would be worth improving this bit. Or maybe we can just get rid of all the remaining `redux-bridge` usages 😉